index.md

```diff
@@ -13,107 +12,166 @@ First we set up & capture the content, then we render it in the scaffolding belo
 
 
 {% capture intro-left %}
-## ![Cockpit](/images/site/cockpit-logo.svg)
-{:.logo}
-
-<span aria-hidden="true" role="presentation">Cockpit</span> is a server manager that makes it easy to administer your GNU/Linux servers via a web browser.
-
+Cockpit makes it easy to administer your GNU/Linux servers via a web browser.
 {% endcapture %}
 
 {% capture intro-right %}
 [![Docker screenshot]({{ site.baseurl }}/images/site/screenshot-docker.png)]({{ site.baseurl }}/images/site/screenshot-docker.png){:.screenshot.zoom}
 {% endcapture %}
 
 
-{% capture blurb_easy %}
-[![Storage screenshot]({{ site.baseurl }}/images/site/screenshot-storage.png)]({{ site.baseurl}}/images/site/screenshot-storage.png){:.screenshot.zoom}
-### Easy to use
+{% assign blurbs_highlight = 6 %}
+
+{% capture blurbs %}
+
+## Easy to use
+
+### Discoverable
 Cockpit makes Linux discoverable, allowing sysadmins to easily perform tasks such as starting containers, storage administration, network configuration, inspecting logs and so on.
-{% endcapture %}
 
+[![Storage screenshot]({{ site.baseurl }}/images/site/screenshot-storage.png)]({{ site.baseurl}}/images/site/screenshot-storage.png)
+{:.screenshot.zoom}
+
+### Designed & usability-tested
+Cockpit has been designed with your goals in mind. We also routinely test Cockpit with usability studies to make it work the way you'd expect. As a result, Cockpit gets easier to use all the time.
+
+### Team friendly
+Cockpit is useful for seasoned admins and is friendly enough for those new to Linux. We've designed Cockpit to be friendly for whole teams.
```

Comment:
This makes it sound like it's first and foremost a tool that is good for seasoned admins, and not for those new to Linux. I would reverse the order somehow.